### ISSUE TITLE
How can I set a ChatId as a string?

### ISSUE THREAD

**Issue by frjariello** [schlingensiepen/TelegramBotClient#4]
Hi,
I would like to send messages to a channel that has a ChatId with a larger number than long (-1009999999999 like this), is it possible to set it as a String?
Thanks

P.S.: Ok, I managed to modify it, and now I can also send to the channels with big ChatId

**Comment by frjariello** [comment]
Sorry, after a bit of testing, I suppose the problem is not the ChatId, but the possibility to write in Chanel

**Comment by joern-schlingensiepen** [comment]
Well, this is a real problem, thanks for your idea.

Best Regards

**Comment by sunbit** [comment]
Yep i just had that problem and tried to modify it with a `long long` integer type, but this has a number of problems with either the Serial.print and the json library, at the end i did a quick change to make startPosting public and use it from my code to build the json string manually.
